### ISSUE TITLE
Update to redmine 3.3

### ISSUE THREAD

**Issue by meron1122** [stgeneral/redmine-progressive-projects-list#65]
Menu in project window is offset to right, because redmine add "+" in version 3.3
![zrzut ekranu z 2016-06-24 13 52 21](https://cloud.githubusercontent.com/assets/2895090/16337119/45865c52-3a13-11e6-89c3-404dd22900ca.png)

**Comment by stgeneral** [comment]
Looks like you use custom theme. Do you experience same issue with the default Redmine theme?

Thank you!

**Comment by meron1122** [comment]
On stock theme too. :)

2016-06-24 14:47 GMT+02:00 Dmitry Babenko <notifications@github.com>:

**Comment by stgeneral** [comment]
Ok, will take a look when we'll migrate to Redmine 3.3 :)

**Comment by AndrewRH** [comment]
Bump :)  I tried to fix it but my Ruby knowledge is really bad...If I manage to figure out what's going on then I'll submit a patch...  It'd be great to get his fixed!  I can confirm that with a fresh redmine (no themes/plugins) that the layout is as in the screenshot above.  Looks like a great plugin, can't wait to use it!
Thanks,

**Comment by archonwang** [comment]
+1 same problem..
my environment 

> Environment:
>   Redmine version                3.3.2.stable.16384
>   Ruby version                   2.2.4-p230 (2015-12-16) [x86_64-linux]
>   Rails version                  4.2.7.1
>   Environment                    production
>   Database adapter               Mysql2

@stgeneral Could you make some patch to fix it? Thanks.

**Comment by stgeneral** [comment]
Thank you for the feedback. Will try to take a look at this issue when will have some time. But can't promise any timeline.

**Comment by recordables** [comment]
Just ran into this issue as well.. anyone figured this one out?

**Comment by stgeneral** [comment]
As an temporary workaround you may try to turn off the `+` button in `Settings -> Display -> Project menu tab for creating new objects`.

Thank you for using the plugin and sorry you experience this issue.

**Comment by Saylrman64** [comment]
I had the same issues and for me it came down to some css changes.
I suggest making the changes in your custom theme sytlesheet so that it doesn't interfere with the redmine core.
Also, if you try to change the plugin stylesheet it will get overwritten when you update the plugin.

If you must put the changes in your redmine core application.css or in the plugin .css, be sure you have good notes and you've documented you changes somewhere else so that when you update either redmine or the plugin, your changes can be recreated.

Here is what I changed:
```
/* Progressive Project Plugin page - table and progress width changes */
#projects-index ul.projects div.root a.project {
        font-family: "Lucida Grande", verdana, arial, helvetica, sans-serif;
}

.version-overview table.progress {
        width: 65%;
}

div#projects-index table.progress {
        width: 65% !important;
}

#projects-index ul.projects li.child {
        margin-top: 1.3em;
}

#projects-index ul.projects div.root a.project {
       font-size: 16px;
}

#projects-index ul.projects li.root {
       margin-bottom: 3em !important;
}

.progressive-project-version {
       margin-top: 20px;
}

```

One of the key items was changing the margin top and bottom.

Every theme will be different but once you find the combo of nudges to the margins and spacing it will all look fine.

Hope this helps, good luck.

**Comment by stgeneral** [comment]
Fixed in #90.
